### PR TITLE
customization modal fix

### PR DESCRIPTION
**Pull request by JBLach**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | i have fixed the customization modal opening, code refactoring, and corrected the style
| Type?             | bug fix, refacto
| BC breaks?        |  no
| Deprecations?     |  no
| Fixed ticket?     | Fixes NA.
| How to test?      | see screenshots below.
| Possible impacts? | Please indicate what parts of the software we need to check to make sure everything is alright.

## Before 
image
image

## After

image
image

### CODE REVIEW

File: templates/catalog/_partials/customization-modal.tpl [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+                  height="100"
+                >
+              {/if}
+            </div>
+          </div>
+        {/foreach}
+      </div>
+    </div>
+  </div>
+</div>
```
Comment (NeOMakinG) [inline templates/catalog/_partials/customization-modal.tpl:42]: Missing blank line at the end of the file

File: templates/customer/_partials/order-detail-return.tpl [hunk 2]
```diff
@@ -43,42 +43,10 @@
                 {foreach from=$product.customizations item="customization"}
                   <div class="customization">
                     <a href="#" data-bs-toggle="modal"
-                      data-target="#product-customizations-modal-{$customization.id_customization}">{l s='Product customization' d='Shop.Theme.Catalog'}</a>
+                      data-bs-target="#product-customizations-modal-{$customization.id_customization}">{l s='Product customization' d='Shop.Theme.Catalog'}</a>
```
Comment (NeOMakinG) [inline templates/customer/_partials/order-detail-return.tpl:46]: 🐐

**Review by NeOMakinG — APPROVED** [review]
One small think, but great 🍬

**Review by kpodemski — APPROVED** [review]

**Comment by NeOMakinG** [comment]
Thanks @JBLach

**Pull request merged**